src/mlpack/core/math/random.hpp

```diff
@@ -41,7 +41,12 @@ inline void RandomSeed(const size_t seed)
 {
   #if (!defined(BINDING_TYPE) || BINDING_TYPE != BINDING_TYPE_TEST)
     randGen.seed((uint32_t) seed);
-    srand((unsigned int) seed);
+    # if(BINDING_TYPE == BINDING_TYPE_R)
+      // To suppress Found ‘srand’, possibly from ‘srand’ (C).
+      (void)((unsigned int) seed);
```

Comment:
```suggestion
      (void) seed;
```
I think this will avoid the warning too.  Correct me if I'm wrong. :)

Reply:
This works great.